executor/src/witgen/jit/function_cache.rs

```diff
@@ -90,12 +96,24 @@ impl<'a, T: FieldElement> FunctionCache<'a, T> {
         mutable_state: &MutableState<'a, T, Q>,
         cache_key: &CacheKey,
     ) -> Option<WitgenFunction<T>> {
-        log::trace!("Compiling JIT function for {:?}", cache_key);
+        log::debug!(
+            "Compiling JIT function for \n  Machine: {}\n  Identity: {}\n   Known args: {:?}",
+            self.machine_name,
+            self.parts.connections[&cache_key.identity_id],
+            cache_key.known_args
```

Comment:
IIRC this is just a bitvec, would be better to take the actual arguments from the connection.

Reply:
Yeah, I think it's not perfect but still quite informative :)